src/html-bundler.ts

```diff
@@ -43,8 +44,8 @@ export class HtmlBundler {
     this.document = await this._prepareBundleDocument();
     let ast = clone(this.document.parsedDocument.ast);
     dom5.removeFakeRootElements(ast);
-    this._injectHtmlImportsForBundle(ast);
     this._rewriteAstToEmulateBaseTag(ast, this.assignedBundle.url);
+    this._injectHtmlImportsForBundle(ast);
```

Comment:
was this re-ordering significant?

Reply:
It was not. It turns out to be irrelevant since the URLs as injected are resolved so the rewriteAstToEmulateBaseTag has no effect on the injected HTML import link tags.

However, I have moved the inject back to before the rewrite just because.